packages/api/src/controllers/asset.ts

```diff
@@ -383,7 +388,8 @@ app.put("/upload/:url", async (req, res) => {
           },
         },
         undefined,
-        asset
+        asset,
+        taskId
```

Comment:
We should take care of this as it conflicts with this #895 on [L141](https://github.com/livepeer/livepeer-com/blob/4eb6e2dfa6912fc15c351fb30a5706663f4e3e89/packages/api/src/task/scheduler.ts#L141)?

Reply:
cool thanks!